resource/reactivedrop_german.txt

```diff
@@ -87,13 +87,13 @@
 "[english]asw_SynTekTurret2"		"Turret 2"
 "asw_SynTekTurret3"		"Geschützturm 3"
 "[english]asw_SynTekTurret3"		"Turret 3"
-"asw_turret_text_top_left"		"KOLONIE-SICHERHEITSCHECK\nVERWENDE CODE FÜR MANUELLEN EINGRIFF ALPHA 5/34\n\nINTERNER CHECK DER GESCHÜTZTURMSYSTEME\n0X83 0x32 0x17 0x02 0x82\n0x18 0x53 0x5A 0x70 0x12\n\nZIEL-SICHERHEITSCHECK:\nDEAKTIVIERT"
+"asw_turret_text_top_left"		"KOLONIE-SICHERHEITSSYSTEM\nÜBERBRÜCKUNGSCODE ALPHA 5/34 WIRD AUSGEFÜHRT\n\nÜBERPRÜFUNG DER INTERNEN GESCHÜTZTURMSYSTEME\n0X83 0x32 0x17 0x02 0x82\n0x18 0x53 0x5A 0x70 0x12\n\nZIELSICHERHEITSPRÜFUNG:\nDEAKTIVIERT"
 "[english]asw_turret_text_top_left"		"COLONY SECURITY SYSTEM\nRUNNING OVERRIDE CODE ALPHA 5/34\n\nINTERNAL TURRET SYSTEMS CHECK\n0X83 0x32 0x17 0x02 0x82\n0x18 0x53 0x5A 0x70 0x12\n\nTARGET SAFETY CHECK:\nDISABLED"
-"asw_turret_text_top_right"		"FERNGESTEUERTES SICHERHEITSSYSTEM V264.3C\n\nZUR VERFÜGUNG GESTELLT VON SYNTEK SECURITY, EINE\nVOLLSTÄNDIGE TOCHTERGESELLSCHAFT VON\nSYNTEK MEGACORPORATION, INC.\n\nZIELVERFOLGUNG AKTIV\nGESCHÜTZLAUF: OK"
+"asw_turret_text_top_right"		"FERNÜBERWACHUNGSSYSTEM V264.3C\n\nZUR VERFÜGUNG GESTELLT VON SYNTEK SECURITY, EIN\nHUNDERTPROZENTIGES TOCHTERUNTERNEHMEN VON\nSYNTEK MEGACORPORATION, INC.\n\nZIELVERFOLGUNG AKTIV\nGESCHÜTZLAUF: OK"
 "[english]asw_turret_text_top_right"		"REMOTE SECURITY SYSTEM V264.3C\n\nPROVIDED BY SYNTEK SECURITY, A\nWHOLLY OWNED SUBSIDIARY OF\nSYNTEK MEGACORPORATION, INC.\n\nTRACKING ACTIVE\nFIRING CHAMBER: OK"
 "asw_SynTekMail"		"E-Mail"
 "[english]asw_SynTekMail"		"Mail"
-"asw_SynTekMailInbox"		"Empfangen"
+"asw_SynTekMailInbox"		"Posteingang"
```

Comment:
Die meisten aktuellen Mailprogramme nennen den Eingang so. Allerdings möchte ich nicht unbedingt diese etwas eigenartige, nicht mit "Gesendet" konsistenten Wortwahl die historisch gewachsen ist in ein Sci-Fi Setting übertragen das gewachsene Fehler noch immer besitzt.